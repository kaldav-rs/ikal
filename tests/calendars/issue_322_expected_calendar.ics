BEGIN:VCALENDAR
VERSION:2.0
PRODID:collective/icalendar
BEGIN:VEVENT
DTSTAMP:20221108T080105Z
DTSTART:20220101
UID:636a0cc1dbd5a1667894465@icalendar
SUMMARY:Event with bare string as argument for categories
CATEGORIES:Lecture
END:VEVENT
END:VCALENDAR
