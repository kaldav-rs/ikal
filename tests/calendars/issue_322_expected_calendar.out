Ok(
    VCalendar {
        prodid: Text {
            params: {},
            text: "collective/icalendar",
        },
        version: Text {
            params: {},
            text: "2.0",
        },
        calscale: None,
        method: None,
        alarms: [],
        events: [
            VEvent {
                dtstamp: Local(
                    2022-11-08T09:01:05+01:00,
                ),
                uid: Text {
                    params: {},
                    text: "636a0cc1dbd5a1667894465@icalendar",
                },
                dtstart: Date(
                    2022-01-01,
                ),
                class: None,
                created: None,
                description: None,
                geo: None,
                last_modified: None,
                location: None,
                organizer: None,
                priority: None,
                sequence: None,
                status: None,
                summary: Some(
                    Text {
                        params: {},
                        text: "Event with bare string as argument for categories",
                    },
                ),
                transp: None,
                url: None,
                recurid: None,
                rrule: None,
                dtend: None,
                duration: None,
                attach: [],
                attendee: [],
                categories: [
                    Text {
                        params: {},
                        text: "Lecture",
                    },
                ],
                comment: [],
                contact: [],
                exdate: [],
                rstatus: [],
                related_to: [],
                resources: [],
                rdate: [],
                alarms: [],
                x_prop: {},
                iana_prop: {},
            },
        ],
        freebusy: [],
        journals: [],
        todo: [],
        timezones: [],
        x_prop: {},
        iana_prop: {},
    },
)
